pkgs/development/libraries/ada-url/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub, cmake, cxxopts }:
+
+stdenv.mkDerivation rec {
+  pname = "ada";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "ada-url";
+    repo = "ada";
+    rev = "v${version}";
+    hash = "sha256-jIXZ8TSLMucwvZc+PMQvNBjvhv2S/0Dh6qTTazYv90A=";
+  };
+
+  postPatch = ''
+    substituteInPlace tools/CMakeLists.txt \
+      --replace "import_dependency(cxxopts" "#import_dependency(cxxopts" \
+      --replace "add_dependency(cxxopts)" "find_package(cxxopts REQUIRED)"
+  '';
```

Comment:
I recommend not merging this pull request until `2.2.0` where this issue is resolved: https://github.com/ada-url/ada/pull/341

Reply:
I just released `v2.2.0`. Can you update your PR @marsam?